index.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <!-- the wraping container -->
+    <div class="parent-container">
+      <div class="header-section">
+        <!--under this are my logo and menu-->
+        <h1 class="logo">My Logo</h1>
+        <div class="humberger-menu">
+          <a href="#"><i class="fa fa-bars fa-lg"></i></a>
+        </div>
+      </div>
```

Comment:
- [ ] Good job so far 💪. However, why don't you use a `header` tag to wrap the header section instead of a `div`? that would improve the HTML structure and make the page more accessible.